src/renderer/sw_engine/tvgSwShape.cpp

```diff
@@ -331,8 +389,13 @@ static SwOutline* _genDashOutline(const RenderShape* rshape, const Matrix* trans
                 break;
             }
             case PathCommand::MoveTo: {
-                if (rshape->stroke->trim.individual) _dashMoveTo(dash, pts);
-                else _dashMoveTo(dash, offIdx, offset, pts);
+                if (trimmed) {
+                    if (simultaneous) {
+                        length = _outlineLength(rshape, pts - startPts, cmds - startCmds, true);
```

Comment:
```
_trimPattern(&dash, rshape, _outlineLength(rshape, 0, 0, simultaneous));
```

Reply:
I guess you've made copy-paste, I changed it to be:
```_trimPattern(&dash, rshape, _outlineLength(rshape, pts - startPts, cmds - startCmds, true));```

Reply:
Please remove if-else condition:
```
_trimPattern(&dash, rshape, _outlineLength(rshape, pts - startPts, cmds - startCmds, simultaneous));
```